controllers/users/users.authentication.js

```diff
@@ -71,6 +72,25 @@ var signout = function(req, res) {
 	res.redirect('/#/userManagement');
 };
 
+var changepassword = function(req, res, next) {
+	passport.authenticate('change-password', function(err, user, info) {
+		if (err)
+			throw err;
+		else if (user) {
+			var newPassword = xss(req.body.newPassword);
```

Comment:
As I mentioned below, all this method should do is set the password and call save().then(function(user), function(err)). Personally, I don't think we need a changePassword method in the model at all.